protocol/bc/asset.go

```diff
@@ -45,23 +45,31 @@ func ComputeAssetID(prog []byte, initialBlockID Hash, vmVersion uint64, data Has
 	return def.ComputeAssetID()
 }
 
+func (a *AssetID) ReadFrom(r io.Reader) (int64, error) {
```

Comment:
Can we hold off on adding these methods until AssetID is a protobuf? (Next PR right?)